spec/controllers/home_controller_spec.rb

```diff
@@ -1,10 +1,13 @@
-require "rails_helper"
+require 'rails_helper'
 
 RSpec.describe HomeController, type: :controller do
-  describe "GET #index" do
-    it "returns http success" do
-      get :index
-      expect(response).to have_http_status(:success)
+  describe 'the landing page' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.